agent/connect/ca/provider_vault_test.go

```diff
@@ -1118,23 +1326,8 @@ func getIntermediateCertTTL(t *testing.T, caConf *structs.CAConfiguration) time.
 	return dur
 }
 
-func testVaultProviderWithConfig(t *testing.T, isPrimary bool, rawConf map[string]interface{}) (*VaultProvider, *TestVaultServer) {
```

Comment:
I inlined `testVaultProviderWithConfig` so that we can create the token in between the two steps.